src/libpmemobj/tx.h

```diff
@@ -53,6 +53,8 @@ extern "C" {
 
 #define TX_ALIGN_SIZE(s, amask) (((s) + (amask)) & ~(amask))
 
+#define TX_ULOG_MAX_OVERHEAD 64
```

Comment:
It's 16 bytes generally. But potentially 64 bytes for undo logs due to alignment requirements. I think it's an important distinction.